TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8332,13 +8333,28 @@ sqlLocks: ({ string.Equals(CurrentToken.Text, "LOCKS", System.StringComparison.O
 onRollbackRetain: SQL_ON SQL_ROLLBACK sqlRetain;
 savepointStatement : SQL_SAVEPOINT (savepoint_name=UserDefinedWord) SQL_UNIQUE? onRollbackRetain sqlCursors (onRollbackRetain sqlLocks)?;
 
-
 dropTableStatement: SQL_DROP SQL_TABLE tableOrAliasName;
 tableOrAliasName: tableOrViewOrCorrelationName;
 date: ({ string.Equals(CurrentToken.Text, "DATE", System.StringComparison.OrdinalIgnoreCase) }? KeywordDATE=UserDefinedWord);
 time: ({ string.Equals(CurrentToken.Text, "TIME", System.StringComparison.OrdinalIgnoreCase) }? KeywordTIME=UserDefinedWord);
 timestamp: ({ string.Equals(CurrentToken.Text, "TIMESTAMP", System.StringComparison.OrdinalIgnoreCase) }? KeywordTIMESTAMP=UserDefinedWord);
 datetime_constant: (date | time | timestamp) AlphanumericLiteral;
+
+//TODO Complete TargetVariable , sqlVariable and sqlExpression
+sqlVariable: hostVariable;
+targetVariable: sqlVariable;
```

Comment:
I don't think we are speaking of the same thing. `sqlVariable` in our ANTLR grammar is an abstract construc for all variable types in SQL and currently we only have host variables defined. SQL variable (as in SQL-variable-name) is not defined yet.

Reply:
Ok in this case, I suggest the following:
```suggestion
//Regroup all variables used in a SQL context
//See https://www.ibm.com/docs/en/db2-for-zos/12?topic=elements-variables
//Parameter markers variables are out of scope here, because it seems related only to PREPARE statement
sqlVariable: hostVariable; //TODO global-variable-name | session-variable-name | SQL-parameter-name | SQL-variable-name | transition-variable-name
targetVariable: sqlVariable; //TODO can session-variable-name be used in SET statement ?
```